src/sentry/release_health/duplex.py

```diff
@@ -209,7 +200,7 @@ def compare_ratios(
         return f"sessions ERROR, Invalid field {path} value = {sessions}, from sessions, only positive values are expected. "
     if sessions == metrics == 0.0:
         return None
-    if float(abs(sessions - metrics)) / max(metrics, sessions) > 0.01:
+    if sessions != metrics:
```

Comment:
I think we should keep an epsilon for comparing ratios and other floating points because they will never be equal bitwise.

Reply:
oops accidentally merged .. can put up a fix

Reply:
oops accidentally merged .. can put up a fix

Reply:
@jjbayer not entirely sure, either:

1. we return float from clickhouse as-is (durations)
2. we don't have floats in clickhouse (session/user counts) and probably do the same exact float arithmetic on the results (divide crashing by non-crashing)

Reply:
We have subtle differences like
```
crash_free_rate = 100 - (row["sessions_crashed"] / totals) * 100
```
vs
```
crash_free_rate = 1.0 - (crash_count / total_session_count)
crash_free_rate = 100 * max(0.0, crash_free_rate)
```

but let's just see what happens.

Reply:
just in case https://github.com/getsentry/sentry/pull/31418